src/drivers/driver.ts

```diff
@@ -1619,13 +1637,13 @@ export abstract class CMakeDriver implements vscode.Disposable {
     // Load up kit or presets before starting any drivers.
     if (useCMakePresets) {
       if (configurePreset) {
-        await this.setConfigurePreset(configurePreset);
+        await this._setConfigurePreset(configurePreset);
```

Comment:
>_setConfigurePreset [](http://example.com/codeflow?start=19&length=19)

It seems we have two functions with basically the same name. I don't think an underscore helps differentiate. It seems there are more functions following this pattern and I don't want us to fall into this trap again.  Can we document the underscore ones, or at least open an issue to fix/rename these?

Reply:
#2069